docs/hardware/babble.mdx

```diff
@@ -0,0 +1,56 @@
+# Project Babble (Face Tracking)
+
+## Summary
+
+[Project Babble](https://github.com/SummerSigh/ProjectBabble) is an open-source software project aiming to bring accessible face-tracking to PCVR headsets.
```

Comment:
Consider adding a note/warning here that babble is only for mouth tracking and does not support eye tracking

Reply:
Resolved in #[c2063f9](https://github.com/VRCFaceTracking/docs/pull/15/commits/c2063f9a32e0e1e23ea8fa10e6a546cc599908c9).